e2e/testsuite/testconfig.go

```diff
@@ -52,6 +54,9 @@ const (
 	// defaultRlyTag is the tag that will be used if no relayer tag is specified.
 	// all images are here https://github.com/cosmos/relayer/pkgs/container/relayer/versions
 	defaultRlyTag = "latest"
+
+	// TODO: local is just a dev image that needs to change.
```

Comment:
should this be linked as an issue?

Reply:
will create issues for all the TODOs and link them 👍